### PR TITLE
SEQNG-49 Load Sequence from from UI

### PR DESCRIPTION
**Pull request by jdnavarro**
Please make sure the load sequence function stub has the correct type signature. If that's the type signature loading the sequence from the UI is trivial.

### CODE REVIEW

File: modules/edu.gemini.seqexec.web/edu.gemini.seqexec.web.server/src/main/scala/edu/gemini/seqexec/web/server/http4s/SeqexecUIApiRoutes.scala [hunk 1]
```diff
@@ -103,7 +103,8 @@ class SeqexecUIApiRoutes(auth: AuthenticationService, q: engine.EventQueue) exte
           } yield (obsId, s)
 
           r match {
-            case \/-((i, s)) => Ok(List(Sequence(i.stringValue(), SequenceState.NotRunning, "F2", s.toSequenceSteps, None)))
+            case \/-((i, s)) => SeqexecEngine.load(i) *>
```
Comment (cquiroz) [inline modules/edu.gemini.seqexec.web/edu.gemini.seqexec.web.server/src/main/scala/edu/gemini/seqexec/web/server/http4s/SeqexecUIApiRoutes.scala]: I'm not sure what the `load` call does. I'd think you want the output to be used in the response, though it is a `Unit`

Comment (jdnavarro) [inline modules/edu.gemini.seqexec.web/edu.gemini.seqexec.web.server/src/main/scala/edu/gemini/seqexec/web/server/http4s/SeqexecUIApiRoutes.scala]: @jluhrs can correct me, but from what I understood, the idea for `load` is to set the initial state in the engine which will send the first output through the websocket.

Comment (cquiroz) [inline modules/edu.gemini.seqexec.web/edu.gemini.seqexec.web.server/src/main/scala/edu/gemini/seqexec/web/server/http4s/SeqexecUIApiRoutes.scala]: I think the idea is correct but somewhat I'd expect `load` to do both. read the sequence from the odb and build the internal sequence, so we get away of `ExecutorImpl` altogether

Comment (cquiroz) [inline modules/edu.gemini.seqexec.web/edu.gemini.seqexec.web.server/src/main/scala/edu/gemini/seqexec/web/server/http4s/SeqexecUIApiRoutes.scala]: The original design han  the load response only going through plain HTTP. Why would we want it on the socket too?

File: modules/edu.gemini.seqexec.server/src/main/scala/edu/gemini/seqexec/server/SeqexecEngine.scala [hunk 2]
```diff
@@ -78,6 +78,9 @@ object SeqexecEngine {
     )
   )
 
+  // Stub for Sequence loading.
+  def load(oid: SPObservationID): Task[SeqexecFailure \/ Unit] = ???
```
Comment (jluhrs) [inline modules/edu.gemini.seqexec.server/src/main/scala/edu/gemini/seqexec/server/SeqexecEngine.scala]: It should be
```
def load(q: engine.EventQueue, seqId: SPObservationID): Task[SeqexecFailure \/ Unit]
```

**Pull request merged**